tests/bluetooth/tester/src/btp/btp_gap.h

```diff
@@ -319,6 +319,24 @@ struct btp_gap_padv_sync_transfer_recv_cmd {
 	uint8_t flags;
 } __packed;
 
+#define BTP_GAP_CONNECT_BR				0x2a
+struct btp_gap_connect_br_cmd {
+	bt_addr_t address;
+} __packed;
+
+
+#define BTP_GAP_PAIR_LEVEL_0		0
+#define BTP_GAP_PAIR_LEVEL_1		1
+#define BTP_GAP_PAIR_LEVEL_2		2
+#define BTP_GAP_PAIR_LEVEL_3		3
+#define BTP_GAP_PAIR_LEVEL_4		4
```

Comment:
Are these and the ones for L2CAP introduced in this PR not always the same as the `BT_SECURITY_L*` values? Do we need to redefine them?

Reply:
Thank you! Yes, they are same. I have thought about this problem before.

From the perspective of the BTP protocol, this value is defined by the BTP protocol. I am not sure if there is a specification similar to the Bluetooth assign number for BTP. If there is, I can define this security level there. Then, GAP and L2CAP can share it. However, I have not found it yet. 

From the perspective of the current BTP code implementation, I have not found a shareable place to define this security level.

What is your suggestion? Thank a lot.

Reply:
Updated.